package.json

```diff
@@ -33,6 +37,7 @@
     "systemjs": "^0.19.6",
     "systemjs-builder": "^0.14.11",
     "ts-node": "^0.5.4",
+    "tsd": "^0.6.5",
```

Comment:
the repo should use `typings`. `tsd` will be deprecated soon. (you also want to `.gitignore` `typings/` folder)